images/syncer/Dockerfile

```diff
@@ -54,4 +54,6 @@ RUN mkdir -p config
 
 ADD pkg/apis/cnsoperator/config/cnsregistervolume_crd.yaml /config/
 
+ADD pkg/internal/cnsoperator/config/cns.vmware.com_cnsfilevolumeclients.yaml /config/
```

Comment:
Why do we need this prefix: `cns.vmware.com_` for the yaml ?
Can it be removed. Looks like other crd yamls do not have prefix. Let's keep it consistent ?

Reply:
done